<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WMain</class>
 <widget class="QMainWindow" name="WMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>386</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>386</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>386</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Katakana Tables</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gb_columns">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string> Letters </string>
    </property>
    <widget class="QCheckBox" name="cb_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>n</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>w-</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>r-</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>y-</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>m-</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>h-</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>n-</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>t-</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>s-</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>k-</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>131</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string> Table </string>
    </property>
    <widget class="QSpinBox" name="sb_columns">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="value">
      <number>12</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_rows">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="value">
      <number>18</number>
     </property>
    </widget>
    <widget class="QLabel" name="l_columns">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Columns:</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_rows">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Rows:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="b_generate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>New table</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="t_tables">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>441</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="t_task">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Task</string>
     </attribute>
     <widget class="QScrollArea" name="sa_task">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>10</y>
        <width>421</width>
        <height>281</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>419</width>
         <height>279</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="b_task_print">
      <property name="geometry">
       <rect>
        <x>342</x>
        <y>300</y>
        <width>87</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Print</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="t_checker">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Checker</string>
     </attribute>
     <widget class="QScrollArea" name="sa_checker">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>10</y>
        <width>421</width>
        <height>281</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>419</width>
         <height>279</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="b_checker_print">
      <property name="geometry">
       <rect>
        <x>342</x>
        <y>300</y>
        <width>87</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Print</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="t_about">
     <attribute name="title">
      <string>About</string>
     </attribute>
     <widget class="QLabel" name="l_about">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>311</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;

&lt;p&gt;&lt;b&gt;Katakana Tables&lt;/b&gt;. &lt;i&gt;Version:&lt;/i&gt; 0.1.&lt;/p&gt;

&lt;p&gt;Katakana Tables is small, tiny and helpful program for those, who have just began study Japanese and need to learn Katakana. By this program it's possible to simple generate and print tables with katakana letters for training.&lt;/p&gt;

&lt;p&gt;If you find any bug, please share information with developer &lt;a href=&quot;https://github.com/Melkogotto/JapanTables/issues&quot;&gt;on GitHub site&lt;/a&gt;.&lt;/p&gt;

&lt;p&gt;Katakana Tables is part of &lt;b&gt;Japan Tables&lt;/b&gt; project, which is OpenSource and licensed under GNU GPL v2. You can find source code also &lt;a href=&quot;https://github.com/Melkogotto/JapanTables&quot;&gt;on GitHub site&lt;/a&gt;.&lt;/p&gt;
&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>b_generate</sender>
   <signal>clicked()</signal>
   <receiver>WMain</receiver>
   <slot>generateTables()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_task_print</sender>
   <signal>clicked()</signal>
   <receiver>WMain</receiver>
   <slot>printTable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_checker_print</sender>
   <signal>clicked()</signal>
   <receiver>WMain</receiver>
   <slot>printChecker()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>generateTables()</slot>
  <slot>printTable()</slot>
  <slot>printChecker()</slot>
 </slots>
</ui>
